### PR TITLE
Budget App v.1.0

### PR DESCRIPTION
**Pull request by MAleemH**
### PR Includes:

- Created rails app with `PostgreSQL` database.
- Created database with validations according to given ERD.
- Added `rubocop` and `stylelint` linters.
- Added following interactions:
- [x] Created splash screen.
- [x] Created sign up and login page.
- [x] Created categories page as home page.
- [x] Created add a category page.
- [x] Created transactions page.
- [x] Created add a transaction page.
- Added `rspec` tests.
- Followed given design.
- Deployed website on `render`.
- [Click here](https://www.loom.com/share/a468402ed7c14a9493d88ec3d21bf881) to watch presentation video.

**NOTE FOR REVIEWER:** Sorry! My webcam was on but it was working slowly, it was still processing during the whole recording and it didn't record my face in full video. It recorded my face only in some parts of the recording. I'll request you please ignore this if you can't see my face in the whole video.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,227 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://budget-app-g6vk.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (emmanuellekamwa) [inline README.md:82]: - [ ] It is a requirement for you to record a five-minute loom video describing the features of the project. Please you have to do that to meet the project requirements.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ❌⛔🛑**
Hello @MAleemH 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive PR.
- Good commit messages.
- App functioning correctly.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ❌⛔🛑**
Hello @MAleemH 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive PR.
- Good commit messages.
- App functioning correctly.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED ❌⛔🛑**
Hello @MAleemH 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Descriptive PR.
- Good commit messages.
- App functioning correctly.

## Required Changes ♻️
_Check the comments under the review._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (MAleemH) [reply README.md:82]: @emmanuellekamwa I have added video link in PR description. Please visit that. There is no requirement to add it in readme file.

Comment (MAleemH) [reply README.md:82]: @emmanuellekamwa Please watch the video, and also mark approve if it is acceptable. Your current review is not appearing in my dashboard. Thanks.

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes Implemented ✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**